dev.yml

```diff
@@ -0,0 +1,23 @@
+name: shopify-api-js
+
+type: node
+
+up:
+  - node:
+      version: v18.19.0
+      yarn: 1.22.21
+      packages:
+        - .
+env:
+  NODE_ENV: "development"
+
+commands:
+  server:
+    run: yarn dev
+  test:
+    syntax:
+      argument: file
+      optional: args...
```

Comment:
Is this right? It looks kind of template-y 😄

Reply:
yeah that's a good point, I think it'll take this out because we'll use yarn for everything, dev.yml is just here for compliance